.buildkite/metricbeat/pipeline.yml

```diff
@@ -10,43 +12,143 @@ env:
   IMAGE_WIN_2019: "family/core-windows-2019"
   IMAGE_WIN_2022: "family/core-windows-2022"
   IMAGE_MACOS_X86_64: "generic-13-ventura-x64"
-  BEATS_PROJECT_NAME: "metricbeat"
+
   GCP_DEFAULT_MACHINE_TYPE: "c2d-highcpu-8"
   GCP_HI_PERF_MACHINE_TYPE: "c2d-highcpu-16"
   GCP_WIN_MACHINE_TYPE: "n2-standard-8"
   AWS_ARM_INSTANCE_TYPE: "t4g.xlarge"
 
-
 steps:
+  - group: "Metricbeat Mandatory Tests"
+    key: "metricbeat-mandatory-tests"
+    steps:
+      - label: ":linux: Metricbeat Ubuntu Unit Tests"
+        key: "metricbeat-mandatory-linux-unit-test"
+        command: ".buildkite/scripts/unit_tests.sh"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_UBUNTU_X86_64}"
+          machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
+        artifact_paths: "${BEATS_PROJECT_NAME}/build/*.*"
+
+      - label: ":go: Metricbeat Go Integration Tests"
+        key: "metricbeat-mandatory-int-test"
+        command: ".buildkite/scripts/go_int_tests.sh"
```

Comment:
We should not be using the wrapper scripts, since we are using our own images.

Reply:
https://github.com/elastic/beats/blob/main/.buildkite/scripts/generate_metricbeat_pipeline.sh#L29 
Probably some changes are not merged yet

Reply:
the migration away of the wrapper scripts is still WIP, the x-pack was refactored not the oss